api/v1alpha1/virtualmachine_types.go

```diff
@@ -43,6 +43,11 @@ const (
 	Unknown VMStatusPhase = "Unknown"
 )
 
+// Annotation label to pause the reconciliation of a VM.  If set, VM operator will not reconcile a VirtualMachine
+// resource.  This can be used when a Virtual Machine needs to be modified out-of-band of VM operator on the
+// infrastructure directly (e.g., during a VADP based Restore operation).
+const PauseReconcileAnnotationLabel = GroupName + "/pause-reconcile"
```

Comment:
What should the label value be? You could mention it in the comment, or introduce a variable/s for possible values.

Reply:
"AnnotationLabel" is very confusing. Seems this is used as an annotation so I'd suggest to drop the Label part.

Reply:
Done

Reply:
> What is the expected label value? You could mention it in the comment, or introduce a variable/s for possible values.

The value of this annotation does not matter. We will only check the presence of the annotation to detect if we need to skip the reconcile.
Ref: CAPI does something similar: 
- https://github.com/kubernetes-sigs/cluster-api/blob/3da7a18b40810cd8a33975e81c89978c6c8beedc/api/v1beta1/common_types.go#L61No
- https://github.com/kubernetes-sigs/cluster-api/blob/58905fb5140a7844c3e24aa70d255fb8f9d0c491/util/annotations/helpers.go#L41